### PR TITLE
Daily blog - add views

### PR DESCRIPTION
**Pull request by jssol**
## Project requirements

- Implemented the design from the sneak peek wireframes.
    - Styling is nice to have but focus on making sure everything functions first.

- Used methods that I have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,11 @@
...
+  <% @users.each do |user| %>
+    <li class="flex items-center gap-2 border-2 p-4">
+      <img class="w-24 h-24" src=<%= user.photo %> alt=<%= user.name %> />
+      <div>
+        <p>Name: <%= user.name %></p>
+        <p>Number of posts: <%= user.posts_counter %></p>
+      </div>
+    </li>
+  <% end %>
+</ul>
```
Comment (ahmetbozaci) [inline app/views/users/index.html.erb:11]: - [ ] A number of posts is wrong. The user has 2 posts that ı create but it shows 6. Kindly fix it.
- [ ] Also ı should be able to see the button on this page. And with a click on the button ı should see all posts. Kindly fix it. [Wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png)
image

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,26 @@
...
+  </div>
+</section>
+<ul class="w-full flex flex-col gap-2 p-4">
+  <% @posts.each do |post| %>
+    <li class="border-2 p-2">
+      <div>
+        <p class="text-md font-semibold">Post #<%= post.id %>: <%= post.title %></p>
+        <p><%= post.text %></p>
+        <div class="w-3/6 flex items-center gap-8">
+          <p>Comments: <%= post.comments_counter %></p>
```
Comment (ahmetbozaci) [inline app/views/posts/index.html.erb:15]: - [x] Comment number is showing wrong. Kİndly fix it.

image
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @jssol  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: app/views/users/index.html.erb (hunk 1 above)
Comment (jssol) [reply app/views/users/index.html.erb:11]: This requirement I believe is for the next milestone when we have to create forms.

Comment (ahmetbozaci) [reply app/views/users/index.html.erb:11]: @jssol Number of posts or number of comments automatically should come from methods that you already create it in old milestone.

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (jssol) [reply app/views/posts/index.html.erb:15]: Thanks, it's done!!

Comment (jssol) [reply app/views/posts/index.html.erb:15]: Done!!

**Review by EricMbouwe — APPROVED** [review]
Hi @jssol, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Good job making changes ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**